python/benchmarks/parquet.py

```diff
@@ -131,3 +131,22 @@ def time_write_random_order(self, nunique):
 
     def time_write_sequential(self, nunique):
         pq.write_table(self.table_sequential, pa.BufferOutputStream())
+
+
+class ParquetManyColumns(object):
+
+    total_cells = 1000000
+    param_names = ('num_cols',)
+    params = [100, 1000, 10000]
+
+    def setup(self, num_cols):
+        num_rows = self.total_cells // num_cols
+        table = pa.table({'c' + str(i): np.random.randn(num_rows)
+                          for i in range(num_cols)})
+
+        out = pa.BufferOutputStream()
+        pq.write_table(table, out)
+        self.buf = out.getvalue()
+
+    def time_read_time(self, num_cols):
+        pq.read_table(self.buf)
```

Comment:
Maybe can directly add a timing for writing the file as well?

```
def time_write(self, num_cols):
    out = pa.BufferOutputStream()
    pq.write_table(self.table, out)
```

?

Reply:
done